app/models/comfy/cms/file.rb

```diff
@@ -23,7 +23,9 @@ class Comfy::Cms::File < ActiveRecord::Base
   # -- Callbacks ---------------------------------------------------------------
   before_validation :assign_label, on: :create
   before_create :assign_position
-  after_save :process_attachment
+  # active_storage attachment behavior changed in rails 6 - see PR#892 for details
+  before_save :process_attachment, if: proc { Rails::VERSION::MAJOR >= 6 }
```

Comment:
Instead of a `proc` that checks this every time, consider simply doing:

```ruby
if Rails::VERSION::MAJOR >= 6
  before_save :process_attachment
else
  after_save :process_attachment
end
```

Reply:
much simpler